content/01.abstract.md

```diff
@@ -1,11 +1,10 @@
 ## Abstract {.page_break_before}
 
-Deep learning, which describes a class of machine learning algorithms, has recently showed impressive results across a variety of domains.
-Biology and medicine are data rich, but the data are complex and often ill-understood.
-Problems of this nature may be particularly well-suited to deep learning techniques.
-We examine applications of deep learning to a variety of biomedical problems---patient classification, fundamental biological processes, and treatment of patients---and discuss whether deep learning will transform these tasks or if the biomedical sphere poses unique challenges.
-We find that deep learning has yet to revolutionize or definitively resolve any of these problems, but promising advances have been made on the prior state of the art.
-Even when improvement over a previous baseline has been modest, we have seen signs that deep learning methods may speed or aid human investigation.
-More work is needed to address concerns related to interpretability and how to best model each problem.
+Deep learning, which describes a class of machine learning algorithms focussing on the training of deep artificial neural networks, has recently shown impressive results across a variety of domains.
```

Comment:
I think we specifically left `artificial neural network` algorithms out of our definitions in many cases. What I think is more important is that these methods can work on raw data to produce intermediate features that are then used for some subsequent task. What about:

> Deep learning describes a class of machine learning algorithms that are capable of learning to combine relatively raw inputs into layers of intermediate features, and such algorithms now perform impressively across a variety of domains.

Reply:
@cgreene that version has good ideas, but I think it would work better as 2 sentences.  How about:

> Deep learning describes a class of machine learning algorithms that are capable of combining raw inputs into layers of intermediate features. These algorithms now perform impressively across a variety of domains.

I'm not sure why, but the longer `recently shown impressive results across a variety of domains` sounds more natural to me.

Reply:
I agree with `recently shown impressive results across a variety of domains`. Another suggestion would be `produces remarkable results across a variety of domains compared to more traditional methods`, but I would prefer the former.

Reply:
Let's go with the more compact `recently shown impressive results across a variety of domains`